### DISCUSSION TITLE
[Feature Request] widgets!

### DISCUSSION THREAD

**Discussion by AuxiliaryPanther** [TylerWilliamson/QuickWeather#172] [Ideas]
Widgets would be amazing with this app. Given the open source nature of the input data, I understand if radar is off the table, but I feel like current weather + forecast would be useful. Thanks!

**Comment by TylerWilliamson** [comment]
Widgets have been asked for since literally the first issue (#1). In the past, I have flatly said "no" to widgets because I feel they will become a can of worms where everyone will want something different and I do not particularly want to go down that route.

However, given the enthusiasm, I will consider adding widgets and leave this discussion opn to discuss them.

That does not mean they will be coming any time soon, though. I have other higher priority items planned and currently, I have little time to code

**Reply by AuxiliaryPanther** [reply]
I too have little time to [learn how to] code, so I do not wish you any stress as you tackle the priorities. The notification bar does work well, even with GrapheneOS!
